__tests__/app.test.js

```diff
@@ -176,6 +176,49 @@ describe('/api/articles', () => {
     })
 })
 
+describe('/api/articles/:article_id/comments', () => {
+    test('GET 200: responds with an array of comments for given article_id with correct properties', () => {
+        return request(app)
+        .get('/api/articles/3/comments')
+        .expect(200)
+        .then(({body: {comments}}) => {
+            expect(comments.length).toBe(2)
+            comments.forEach((comment) => {
+                expect(typeof comment.comment_id).toBe('number')
```

Comment:
Another matcher that may be useful to look into: .toMatchObject()

Reply:
that sounds so much easier than what I did haha